docs/QbeastFormat.md

```diff
@@ -63,15 +63,15 @@ Here you can see the changes on the `AddFile` **`tags`** information
 }
 ```
 
-| Term           | Description                                                                                                                                  |
-|----------------|----------------------------------------------------------------------------------------------------------------------------------------------|
-| `revision`     | The metadata of the tree. Locates the particular tree in the space, which includes the min/max values for each column indexed and timestamp. |
-| `blocks`       | A list of the blocks that are stored inside the file. Can belong to different cubes.                                                         |
-| `cube`         | The serialized representation of the Cube.                                                                                                   |
-| `minWeight`    | The minimum weight of the block.                                                                                                             |
-| `maxWeight`    | The maximum weight of the block.                                                                                                             |
-| `elementCount` | The number of elements in the block.                                                                                                         |
-| `replicated`   | A flag that indicates if the block is replicated. Replication is **not available** from v0.6.0.                                                  |
+| Term           | Description                                                                                                    |
+|----------------|----------------------------------------------------------------------------------------------------------------|
+| `revision`     | **Locates a particular `min`-`max` range of the columns** in the space, and includes a Timestamp of the creation time. |
+| `blocks`       | A **list of the blocks** that are stored inside the file. Can belong to different cubes.                           |
```

Comment:
The concept of blocks is a bit weird. Blocks at least in storage are addressable, like I can fetch a block from a disk drive by providing a block number. I'm under the impression that we cannot really address a block directly, in which case, it only serves the purpose of indicating that a file contains data from a particular cube?

Reply:
Ideally, we should be able to retrieve only the blocks from a certain file that contains data that satisfies the query. The functionality is not implemented, though it requires some #175 support to be able to discard row groups from Parquet on the fly.

Reply:
Ok, seems good to me